data/notices.json

```diff
@@ -727,6 +727,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(credentials): CLI fails to validate credentials under certain conditions",
+      "issueNumber": 32653,
+      "overview": "Credential Providers returning empty credentials, or credentials with null expiration fields, would cause the CLI to throw an exception from CDK versions 2.172.0 until 2.173.1.",
```

Comment:
```suggestion
      "overview": "Credential Providers returning empty credentials, or credentials with null expiration fields, would cause the CLI to throw an exception. If you encounter this behavior, upgrade to version 2.173.2 or higher",
```